index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <main>
+        <header class="header"> 
+            <a href="#">Bwalya Nonde</a>
+            <nav class="nav-menu">
+                <img src="images/union.png" alt="">
```

Comment:
Please add a hamburger button on the right side of the header section. This button does not show on the page. Kindly fix it.
![hamburger](https://user-images.githubusercontent.com/35504149/164798615-ebe621b0-1232-499d-8549-f91fd890c86a.png)